theme/js/script.js

```diff
@@ -119,13 +119,25 @@ function addActiveClassToNavbar() {
             }
         }
     });
-};
-
+}
 function postCardComponentJsonData() {
-    $.getJSON('theme/json/posts.json', function(data) { 
+    $.getJSON('theme/json/posts.json', function(data) {
         $.each(data, function(i, post) {
             $('.card-component-container .row-container--horizontal')
-            .append('<div class="card-wrapper"><div class="card card-one clickable card--horizontal"><div class="card-image-wrapper"><img src="'+post.imageLink+'" alt="'+post.imageAltText+'" class="card-image responsive-image"></div><div class="card-text-container"><h3>'+post.title+'</h3><p>'+post.date+'</p><p>'+post.body+'</p><a href="'+post.link+'">'+post.linkText+'</a></div></div></div>');
-        }); 
-    }); 
-};
+                .append('<div class="card-wrapper"><div class="card card-one clickable card--horizontal"><div class="card-image-wrapper"><img src="'+post.imageLink+'" alt="'+post.imageAltText+'" class="card-image responsive-image"></div><div class="card-text-container"><h3>'+post.title+'</h3><p>'+post.date+'</p><p>'+post.body+'</p><a href="'+post.link+'">'+post.linkText+'</a></div></div></div>');
+        });
+    });
+}
+function slideToggleAccordionContent() {
+    $('.accordion').each(function() {
+        $(this).click(function() {
+            $(this).next('.panel').slideToggle();
+        });
+    });
+}
+
+function addActiveClassToAccordion() {
+    $('.accordion').click(function() {
+        $(this).toggleClass('active');
```

Comment:
You can add this line inside the click function in the `slideToggleAccordionContent()` and then delete this function altogether.

Reply:
done